tutorial/sed_config.yaml

```diff
@@ -1,9 +1,9 @@
 energy:
   calibration:
-    E0: -31.09210508670466
-    d: 1.0954452312102088
+    E0: -196.10237272644352
```

Comment:
I guess these changed as the sed_config is overwritten by the save parameters method

Reply:
This file should have been deleted already anyways. I will remove it here now.